pkgs/tools/security/onlykey-cli/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, python3Packages, stdenv }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "onlykey-cli";
+  version = "1.2.2";
+
+  src = python3Packages.fetchPypi {
+    inherit version;
+    pname = "onlykey";
+    sha256 = "1qkbgab5xlg7bd0jfzf8k5ppb1zhib76r050fiaqi5wibrqrfwdi";
+  };
+
+  # Requires onlykey
```

Comment:
You mean the server or daemon application? Can this easily be added to nix in the future? You don't need to do it now, just curious.

Reply:
```# Requires onlykey``` means: requires having the physical onlykey (a usb security key)